packages/flutter/test/material/tabs_test.dart

```diff
@@ -2213,7 +2214,16 @@ void main() {
     expect(tabBarBox, paints..line(
       color: Colors.white,
     ));
+  });
 
+  testWidgets('Tab indicator color should not be adjusted when disable [automaticIndicatorColorAdjustment]', (WidgetTester tester) async {
+     // Regression test for https://github.com/flutter/flutter/issues/68077
+     final List<String> tabs = <String>['LEFT', 'RIGHT'];
+     await tester.pumpWidget(buildLeftRightApp(tabs: tabs, value: 'LEFT', automaticIndicatorColorAdjustment: false));
```

Comment:
You would also need to apply the `indicatorColor` and `Material.color` to be the same value in the test, since it's expected that otherwise, the color of the indicator is always the expected value

Reply:
This case is the situation that `indicatorColor ` equal `Material.color` because `automaticIndicatorColorAdjustment` is false and the color didn't adjust to white.

This is an opposite test of the above test `Default tab indicator color is white`
```dart
  testWidgets('Default tab indicator color is white', (WidgetTester tester) async {
    // Regression test for https://github.com/flutter/flutter/issues/15958
    final List<String> tabs = <String>['LEFT', 'RIGHT'];
    await tester.pumpWidget(buildLeftRightApp(tabs: tabs, value: 'LEFT'));
    final RenderBox tabBarBox = tester.firstRenderObject<RenderBox>(find.byType(TabBar));
    expect(tabBarBox, paints..line(
      color: Colors.white,
    ));
  });
```